build.zig

```diff
@@ -0,0 +1,26 @@
+const std = @import("std");
+
+pub fn build(b: *std.Build) void {
+    const target = b.standardTargetOptions(.{});
+    const optimize = b.standardOptimizeOption(.{});
+
+    const exe = b.addExecutable(.{
+        .name = "roc",
+        .root_source_file = b.path("src/main.zig"),
+        .target = target,
+        .optimize = optimize,
+    });
```

Comment:
Even though this doesn't have tests yet, can you add this

```suggestion
    const main_path = b.path("src/main.zig");
    const exe = b.addExecutable(.{
        .name = "roc",
        .root_source_file = main_path,
        .target = target,
        .optimize = optimize,
    });
    
    const tests = b.addTest(.{ .root_source_file = main_path });
    const test_cmd = b.addRunArtifact(main_tests);
    
    const test_step = b.step("test", "Run tests");
    test_step.dependOn(&test_cmd.step);

```